packages/cli/src/Studio.ts

```diff
@@ -89,8 +79,6 @@ ${bold('Examples')}
       return this.help(args.message)
     }
 
-    await checkUnsupportedDataProxy('studio', args, true)
```

Comment:
Studio now works with Accelerate URLs. There will be a test, but not via this PR as there is enough content to go through. Consider this an opportunistic removal.